jerry-core/vm/vm.c

```diff
@@ -3312,7 +3312,7 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
 
           if (ecma_is_value_integer_number (left_value))
           {
-            *stack_top_p++ = (~left_value) & (ecma_value_t) (~ECMA_DIRECT_TYPE_MASK);
+            *stack_top_p++ = 0xFFFFFFF0 ^ left_value;
```

Comment:
Isn't this is 0xFFFFFFF0 == (~ECMA_DIRECT_TYPE_MASK)? I would prefer some symbols here.